From 834d40589144b67f4ac7ec17cf7ba34a3391ac7f Mon Sep 17 00:00:00 2001
From: Shammah Chancellor <shammah.chancellor@gmail.com>
Date: Sun, 20 Jun 2021 13:25:05 -0700
Subject: [PATCH 125/196] Finalize reward addresses for first mainnet release
 (#202)

* Updating Bitcoin-ABC's address.
* Update one address to be managed differently.
* Updated Tobias' address

Co-authored-by: Tobias Ruck <ruck.tobias@gmail.com>
---
 src/consensus/addresses_mainnet.h | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/src/consensus/addresses_mainnet.h b/src/consensus/addresses_mainnet.h
index 96af9606e..b05520f74 100644
--- a/src/consensus/addresses_mainnet.h
+++ b/src/consensus/addresses_mainnet.h
@@ -7,9 +7,9 @@ namespace MainNet {
         // Foundation
         {"pzmv0yp3kuwcd2cdv9lpu8nsdmzwud9s0upp4rxwc9"},
         // Bitcoin ABC
-        {"ppppwzhcy6kyyhkjsw0s7vvyzxz73vnwzynkaqjssk"},
-        // Be.Cash
-        {"qpwm9atcqdmr2dutnpp5h0t0hmy0y6k595qg0z5egm"},
+        {"qzu2u8z8ala43ae0009gr8l8lsjjl8599cdwdr9mxq"},
+        // Tobias
+        {"qz6shp4gj0vqe83wuu43n9sjxa9hknqumq0xdtwprp"},
         // Saipan Institute
         {"qrd8dgcmvasdewg2535uzkt9mfhgqztwg504dhsfxc"},
         // Stamp
@@ -28,8 +28,8 @@ namespace MainNet {
         {"qzccavyvf9uwwdyqwsa3txqxr5708rss4qqg8p6an4"},
         // Distro Bot
         {"qp7s3y735fut4vn70tvjakyt6lww4lvr55ljvaxpyn"},
-        // Koush
-        {"qrgqdz7ww5v8yqgu9ns4av5m4t4w7jxgyyxc05z9gu"},
+        // Distro Bot
+        {"qp9cd8u62hzhqq7lz79aeqq3ssgfmyz03vc5c4g0ct"},
     };
 } // namespace MainNet
 } // namespace RewardAddresses
-- 
2.39.5 (Apple Git-154)

